tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/SilentRequestTests.cs

```diff
@@ -109,43 +109,58 @@ public async Task ExpiredTokenRefreshFlowTestAsync()
 
         [TestMethod]
         public void BrokerSilentRequestBrokerRequiredTestAsync()
+        {
+            //Broker is not configured by user but is installed
```

Comment:
These tests are testing the various combinations broker configuration values to simulate different authentication scenarios. These scenarios should result in MSAL using its own internal cache as a fallback because the broker is either not installed or not configured by the user.